services/app-api/prisma/migrations/20230612222743_remove_invalided_by/migration.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN;
+/*
+  Warnings:
+
+  - You are about to drop the column `invalidatedByContractRevisionID` on the `RateRevisionsOnContractRevisionsTable` table. All the data in the column will be lost.
+  - You are about to drop the column `invalidatedByRateRevisionID` on the `RateRevisionsOnContractRevisionsTable` table. All the data in the column will be lost.
```

Comment:
Should we ignore these warnings?

Reply:
These are real warnings but dropping those columns is what we want. They were used in my first implementation of the revision finds